frontend/javascripts/oxalis/geometries/cube.ts

```diff
@@ -99,44 +103,44 @@ class Cube {
       vec(min[0], min[1], max[2]),
       vec(min[0], max[1], max[2]),
       vec(min[0], max[1], min[2]),
-    ];
-    // @ts-expect-error ts-migrate(2339) FIXME: Property 'vertices' does not exist on type 'Geomet... Remove this comment to see the full error message
-    this.crossSections[OrthoViews.PLANE_XY].geometry.vertices = [
+    ]);
+
+    this.crossSections[OrthoViews.PLANE_XY].geometry.setFromPoints([
       vec(min[0], min[1], 0),
       vec(min[0], max[1], 0),
       vec(max[0], max[1], 0),
       vec(max[0], min[1], 0),
       vec(min[0], min[1], 0),
-    ];
-    // @ts-expect-error ts-migrate(2339) FIXME: Property 'vertices' does not exist on type 'Geomet... Remove this comment to see the full error message
-    this.crossSections[OrthoViews.PLANE_YZ].geometry.vertices = [
+    ]);
+    this.crossSections[OrthoViews.PLANE_YZ].geometry.setFromPoints([
       vec(0, min[1], min[2]),
       vec(0, min[1], max[2]),
       vec(0, max[1], max[2]),
       vec(0, max[1], min[2]),
       vec(0, min[1], min[2]),
-    ];
-    // @ts-expect-error ts-migrate(2339) FIXME: Property 'vertices' does not exist on type 'Geomet... Remove this comment to see the full error message
-    this.crossSections[OrthoViews.PLANE_XZ].geometry.vertices = [
+    ]);
+    this.crossSections[OrthoViews.PLANE_XZ].geometry.setFromPoints([
       vec(min[0], 0, min[2]),
       vec(min[0], 0, max[2]),
       vec(max[0], 0, max[2]),
       vec(max[0], 0, min[2]),
       vec(min[0], 0, min[2]),
-    ];
+    ]);
 
     for (const mesh of _.values(this.crossSections).concat([this.cube])) {
       mesh.geometry.computeBoundingSphere();
-      // @ts-expect-error ts-migrate(2339) FIXME: Property 'verticesNeedUpdate' does not exist on ty... Remove this comment to see the full error message
-      mesh.geometry.verticesNeedUpdate = true;
+      mesh.geometry.attributes.position.needsUpdate = true;
     }
 
     this.initialized = true;
-    this.updatePosition(getPosition(Store.getState().flycam));
+    this.updatePositionForCrossSections(getPosition(Store.getState().flycam));
+    this.crossSections[OrthoViews.PLANE_XZ].geometry.computeBoundingSphere();
+    this.crossSections[OrthoViews.PLANE_XZ].geometry.attributes.position.needsUpdate = true;
```

Comment:
Is this really needed, here and if so why only for the XZ plane?

Reply:
good catch. this wasn't intentional, as this is already done by `updatePositionForCrossSections` for all planes.